components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -223,11 +225,15 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
             <div className="flex flex-col gap-2">
               <label className="text-sm font-normal text-light-slate-11">Bio</label>
               <textarea
+                maxLength={255}
                 rows={4}
                 placeholder="Tell us about yourself."
                 className="px-3 py-2 rounded-lg bg-light-slate-4 disabled:cursor-not-allowed "
                 name="bio"
+                value={bio}
+                onChange={(e) => setBio(e.target.value)}
               ></textarea>
+              <div>{bio?.length}/255</div>
```

Comment:
nit(blocking): reduce font size a little and use `p` tag instead of `div`
```suggestion
              <p className="text-sm">{bio?.length}/255</p>
```